### PR TITLE
feat: initialize fbridge module

### PR DESCRIPTION
**Pull request by tkxkd0159**
## Description

closes: #XXXX

## Checklist:

- [x] I followed the [contributing guidelines](https://github.com/Finschia/finschia-sdk/blob/main/CONTRIBUTING.md) and [code of conduct](https://github.com/Finschia/finschia-sdk/blob/main/CODE_OF_CONDUCT.md).
- [ ] I have added a relevant changelog to `CHANGELOG.md`
- [ ] I have added tests to cover my changes.
- [ ] I have updated the documentation accordingly.
- [x] I have updated API documentation `client/docs/swagger-ui/swagger.yaml`

### CODE REVIEW

File: proto/lbm/fbridge/v1/fbridge.proto [hunk 1]
```diff
@@ -0,0 +1,56 @@
...
+  uint64 denominator = 2;
+}
+
+// Role defines the role of the operator, guardian, and judge.
+enum Role {
+  UNSPECIFIED = 0;
+  GUARDIAN    = 1;
+  OPERATOR    = 2;
+  JUDGE       = 3;
+}
```
Comment (170210) [inline proto/lbm/fbridge/v1/fbridge.proto:56]: add blank line

File: proto/lbm/fbridge/v1/query.proto [hunk 2]
```diff
@@ -0,0 +1,125 @@
...
+
+message OperatorsResponse {
+  repeated string operators = 1;
+}
+
+message JudgesRequest {}
+
+message JudgesResponse {
+  repeated string judges = 1;
+}
```
Comment (170210) [inline proto/lbm/fbridge/v1/query.proto:125]: ditto

**Review by 170210 — COMMENTED** [review]
Please check failed test

**Review by jaeseung-bae — DISMISSED** [review]
LGTM

File: proto/lbm/fbridge/v1/tx.proto [hunk 3]
```diff
@@ -0,0 +1,152 @@
...
+  string from = 1;
+  // the address to update the role
+  string target = 2;
+  // the role to be updated
+  // - unspecified : 0, used to remove the address from a group
+  // - guardian : 1
+  // - operator : 2
+  // - judge : 3
+  Role role = 3;
+}
```
Comment (zemyblue) [inline proto/lbm/fbridge/v1/tx.proto:134]: If there is no `seq` field in the `MsgUpdateRole`, I think we cannot distinguish it from the previous proposal when another proposal with the same param is added. 

So how about adding `seq` field or modifying it with another solution?

Comment (tkxkd0159) [reply proto/lbm/fbridge/v1/tx.proto]: I separated voting from submission like you suggested before. So the proposal ID(`seq`) will be assigned automatically.

**Review by zemyblue — APPROVED** [review]

**Review by jaeseung-bae — APPROVED** [review]

**Pull request merged**